tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -278,7 +278,7 @@ export interface uploadResourceArgs {
   option?: string
 }
 
-export const uploadResource = async (args: uploadResourceArgs): Promise<void> => {
+export const uploadResourceAtOnce = async (args: uploadResourceArgs): Promise<void> => {
```

Comment:
IMO, this can be left unchanged
```suggestion
export const uploadResource = async (args: uploadResourceArgs): Promise<void> => {
```